new_platforms/docs/visualstudio_dev.md

```diff
@@ -0,0 +1,109 @@
+Developing your own enclave using Visual Studio
+=============
+
+## Using Open Enclave from Windows
+
+This section covers how to use this SDK to develop your own enclave that can run
+on both SGX and OP-TEE, and use it from your own application.
+
+**Prerequisites:**
+1. Install the Open Enclave Visual Studio Extension
```

Comment:
Where can one get the VSIX package currently? Will you be publishing it to the VS Marketplace?

Reply:
Once it's published, yes.  The md will be updated again at that time (in a future changelist)